packages/utilities/psammead-styles/src/typography.stories.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import {
+  getCanon,
+  getTrafalgar,
+  getParagon,
+  getDoublePica,
+  getGreatPrimer,
+  getBodyCopy,
+  getPica,
+  getLongPrimer,
+  getBrevier,
+  getMinion,
+} from '@bbc/gel-foundations/typography';
+import { latin } from '@bbc/gel-foundations/scripts';
+import TypographyText from '.';
+
+const script = latin;
+const sampleText =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.';
+
+storiesOf('Typography', module)
+  .addDecorator(withKnobs)
+  .add('Canon', () => (
+    <TypographyText script={script} typographyFunc={getCanon}>
```

Comment:
Looks great so far. Adding the inputProvider to switch different scripts via the drop-down list would be great:
e.g. like the Paragraph story example: https://bbc.github.io/psammead/?path=/story/paragraph--default
https://github.com/bbc/psammead/blob/5700a49222fffef327bf73a157f686367b4064e0/packages/components/psammead-paragraph/src/index.stories.jsx#L12

Reply:
👍 Done